src/index.css

```diff
@@ -1802,6 +1802,7 @@ a.altbutton {
 }
 
 a.flatbutton {
+  border-radius: 1rem;
```

Comment:
I think it looks better without better; remove this line.

Reply:
Removed in last commit